metricbeat-tests/runner_test.go

```diff
@@ -56,23 +64,78 @@ func (mts *MetricbeatTestSuite) CleanUp() error {
 	log.WithFields(log.Fields{
 		"service": mts.ServiceName,
 	}).Debug("Service removed from compose.")
+
 	return err
 }
 
+// As we are using an index per scenario outline, with an index name formed by metricbeat-version1-module-version2,
+// and because of the ILM is configured on metricbeat side, then we can use an asterisk for the index name:
+// each scenario outline will be namespaced, so no collitions between different test cases should appear
+func (mts *MetricbeatTestSuite) getIndexName() string {
+	mVersion := strings.ReplaceAll(mts.Version, "-SNAPSHOT", "")
+
+	index := fmt.Sprintf("metricbeat-%s-%s-%s", mVersion, mts.ServiceName, mts.ServiceVersion)
```

Comment:
There would still be the potential problem if we have two scenarios using the same versions? Could we create here a random index name and store it in `mts` to solve this?

Reply:
Yeah, I can send something on top of it